rust/tools/qltest.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+set "RUST_BACKTRACE=full"
+set "QLTEST_LOG=%CODEQL_EXTRACTOR_RUST_LOG_DIR%/qltest.log"
```

Comment:
This looks fine, but do we have any CI check running some qltests on Windows?

Reply:
Well, not yet. I would rather test that boss an integration test though, by adding at least one QL test there